README.md

```diff
@@ -27,165 +27,142 @@ and signs AMP documents as requested by the AMP Cache.
+     signed-exchange)".
+
+#### Demonstrate privacy-preserving prefetch
+
+This step is optional; just to show how [privacy-preserving
+prefetch](https://wicg.github.io/webpackage/draft-yasskin-webpackage-use-cases.html#private-prefetch)
+works with SXGs.
+
+  1. `go get -u github.com/ampproject/amppackager/cmd/amppkg_dl_sxg`.
+  2. `amppkg_dl_sxg https://localhost:8080/priv/doc/https://example.com/`
+  3. Stop `amppkg` with Ctrl-C.
+  4. `go get -u github.com/ampproject/amppackager/cmd/amppkg_test_cache`.
+  5. `amppkg_test_cache`
+  6. Open Chrome and DevTools.
+  7. Visit `https://localhost:8000/`. Observe the prefetch of `/test.sxg`.
+  8. Click the link. Observe that the cached SXG is used.
+
+#### Productionizing
+
+For now, productionizing is a bit manual. We would love to see a Docker
+container that builds in the best practices!
+
+The minimum steps are:
+
+  1. Don't pass `-development` flag to `amppkg`. This causes it to serve HTTP
+     rather than HTTPS.
+  2. Don't expose `amppkg` to the outside world; keep it on your internal
+     network.
+  3. Configure your TLS-serving frontend server to conditionally proxy to
+     `amppkg`, if any of:
+     1. The URL starts with `/amppkg/`.
+     2. The URL is AMP and the `AMP-Cache-Transform` request header is present.
+  4. Configure your frontend server to add `Vary: AMP-Cache-Transform` if the
+     URL is AMP. This should occur for both HTML and SXG responses.
+  5. Get an SXG cert from your CA. It must use an EC key with the prime256v1
+     algorithm, and it must have a [CanSignHttpExchanges
+     extension](https://wicg.github.io/webpackage/draft-yasskin-httpbis-origin-signed-exchanges-impl.html#cross-origin-cert-req).
+     You MUST use this in `amppkg.toml`, and MUST NOT use it in your frontend.
+
+You may also want to:
+
+  1. Launch `amppkg` as a restricted user.
+  2. Save its stdout to a rotated log somewhere.
+
+Once you've done the above, you should be able to launch Chrome without any
+flags; just make sure chrome://flags/#enable-signed-http-exchange is enabled.
+
+#### Replicating
+
+If you need to set up multiple replicas of `amppkg` for scale, you'll want your
+`OCSPCache` to be backed by a shared storage device (e.g. NFS). It doesn't need
+to be shared among all instances globally, but perhaps among all instances per
+datacenter. The reason for this is to reduce the number of OCSP requests
+`amppkg` needs to make, per [OCSP stapling
+recommendations](https://gist.github.com/sleevi/5efe9ef98961ecfb4da8).
+
+#### How will these web packages be discovered by Google?
+
+For now, the presence of the `Vary: AMP-Cache-Transform` response header on an
+AMP HTML page will cause the Google AMP Cache to make a second request with
+`AMP-Cache-Transform: google` for the SXG.
+
+In the future, Googlebot may make all requests with `AMP-Cache-Transform: google`,
+eliminating the second fetch. It would be able to "unwrap" these SXGs and use
+them for Search.
```

Comment:
> It would be able to "unwrap" these SXGs and use them for Search.

For this audience, there isn't enough context to really understand this sentence. I don't think it's needed, I would just drop it.